### ISSUE TITLE
model overfitting issue

### ISSUE THREAD

**Issue by lilyzhng** [rmokady/CLIP_prefix_caption#59]
![image](https://user-images.githubusercontent.com/10191088/203416669-789108d9-e9d4-4425-b07d-22fc55cacbdb.png)
Dear authors,
We have tested the COCO model with different frames, but outputting the same caption results. It seems to be a overfitting issue. Could you suggest if any parameters need to be specified at inference time, or it could be a data issue that the input frames are similar to some extent?

**Comment by arminbazza** [comment]
Hi, did you find a solution to this problem?

**Comment by ret7020** [comment]
@alchemz Can you show your hyperparameters?

**Comment by zzk2021** [comment]
Me too. I have the same problem

**Comment by zzk2021** [comment]
It seems like the model isn‘t train completely. The loss at a high level. the outputs are high-frequency words
![image](https://user-images.githubusercontent.com/73774678/222891853-3987e740-c65a-46a7-b7a6-697454b91680.png)

**Comment by lilyzhng** [comment]
Hi all, unfortunately I haven't find a solution to this. 

@ret7020 For your question, we did not perform hyparameter tuning on the model with new datasets, but simply run inference with the existing model.
